runtime/stdlib/account.go

```diff
@@ -894,7 +894,7 @@ func newAccountKeysRevokeFunction(
 				var err error
 				var accountKey *AccountKey
 				errors.WrapPanic(func() {
-					accountKey, err = handler.RevokeAccountKey(address, index)
+					accountKey, err = handler.RevokeAccountKey(address, uint32(index))
```

Comment:
same here.

Reply:
Fixed in [63abfd5](https://github.com/onflow/cadence/pull/3465/commits/63abfd5ec7b0d11bf142eead83591bf2e6c83ebb)